### ISSUE TITLE
Colon in frontmatter broke build

### ISSUE THREAD

**Issue by oliverjam** [oliverjam/jaredhill-v2#61]
Yo just FYI the latest deployment failed. If you click the "netlify | failed" button on the GitHub homepage it'll take you to the build log that shows what the problem is.

<img width="1033" alt="Screenshot 2021-04-03 at 13 40 07" src="https://user-images.githubusercontent.com/9408641/113478671-2d346580-9482-11eb-903a-cbb405cbb0f1.png">

<img width="1193" alt="Screenshot 2021-04-03 at 13 41 26" src="https://user-images.githubusercontent.com/9408641/113478687-4b01ca80-9482-11eb-91c1-10ce00696be5.png">

Basically the `:` symbol is used to separate the keys from values in the frontmatter data for each post. I.e.
```yaml
title: Designers mind your language
```

If you put a colon in the value itself it that will break the parsing:
```yaml
title: Designers: mind your language
```

You can solve any special character issues like this by wrapping the text in quotes:
```yaml
title: "Designers: mind your language"
```

**Comment by oliverjam** [comment]
@Jamchiller I can have Netlify email you when a deploy fails (since otherwise you'd have to notice on GitHub). Shall I use the address from the contact form?
